.evergreen/config/functions.yml

```diff
@@ -103,7 +103,7 @@ functions:
         content_type: ${content_type|application/x-gzip}
         permissions: public-read
         local_file: ${build_id}.tar.gz
-        remote_file: ${project}/${build_variant}/${revision}/${task_name}/${version}.tar.gz
+        remote_file: mongo-php-driver/${build_variant}/${revision}/${task_name}/${version_id}.tar.gz
```

Comment:
Noted that this fixes the v1.17 Evergreen failures I [mentioned in Slack earlier](https://mongodb.slack.com/archives/C026UA4GF0B/p1700074313027349).

Reply:
I extracted the commit to a separate pull request, which has since been merged.